### PR TITLE
Adding additional parameters needed as step towards multiarm support (Multiarm part 1)

### PR DESCRIPTION
**Pull request by firesurfer**
This PR is part of splitting up into multiple parts #567 

Note:  The non-blocking read won't properly work like this without the PR in this series that allows starting the rtde communication at the latest point of time possible

### CODE REVIEW

**Comment by firesurfer** [comment]
@fmauch @RobertWilbrandt 

I hope you are doing fine! I guess you are quite occupied with other stuff but I would really like to progress on this set of PRs.

File: ur_robot_driver/src/hardware_interface.cpp [hunk 1]
```diff
@@ -281,125 +281,138 @@ URPositionHardwareInterface::on_activate(const rclcpp_lifecycle::State& previous
...
     reverse_ip = "";
   }
-  // Port that will be opened to send trajectory points from the driver to the robot. Note: this feature hasn't been
-  // implemented in ROS2
-  int trajectory_port = 50003;
-  // Port that will be opened to forward script commands from the driver to the robot
-  int script_command_port = stoi(info_.hardware_parameters["script_command_port"]);
-  //  std::string tf_prefix = info_.hardware_parameters["tf_prefix"];
-  //  std::string tf_prefix;
+  //Port of the trajectory interface
```
Comment (fmauch) [inline ur_robot_driver/src/hardware_interface.cpp:308]:
```suggestion

  //Port (on the host pc) of the trajectory interface
```

File: ur_robot_driver/src/hardware_interface.cpp [hunk 2]
```diff
@@ -281,125 +281,138 @@ URPositionHardwareInterface::on_activate(const rclcpp_lifecycle::State& previous
...
+  const int trajectory_port = stoi(info_.hardware_parameters["trajectory_port"]);
```
Comment (fmauch) [inline ur_robot_driver/src/hardware_interface.cpp:309]:
```suggestion

  // Port (on the host PC) that will be used to forward script commands from the driver to the robot
  const int trajectory_port = stoi(info_.hardware_parameters["trajectory_port"]);
```

**Review by fmauch — COMMENTED** [review]
Thank you @firesurfer for your efforts and your patience. I am now back from a 2 months personal break and happy to bring this forward. I Added some minor suggestions regarding comments. Will rebase this and test before merging.

**Comment by fmauch** [comment]
Wanted to re-open it to force CI to run, but apparently I cannot do that.

**Comment by fmauch** [comment]
@firesurfer could you please reopen?

**Pull request closed without merging**